example/package.json

```diff
@@ -63,6 +64,7 @@
     "metro-react-native-babel-preset": "0.73.7",
     "pod-install": "^0.1.38",
     "react-dom": "18.2.0",
+    "react-native-svg-transformer": "^1.0.0",
```

Comment:
I'll be interested to see how this plays out in the `react-native-starter` repo.  Depending on how bad/busted this is _without_ this dev dependency, that could make for a pretty awkward flow.  We want to avoid `react-native-starter` having any knowledge about the SDK beyond peer dependencies.